### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by casellimarco**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+jobs:
+  install-and-test:
+    docker:
+      - image: cimg/python:3.9
+    steps:
+      - checkout
+      - run:
+          name: Install
+          # This assumes pytest is installed via the install-package step above
```
Comment (casellimarco) [inline .circleci/config.yml]:
```suggestion
```

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.PHONY: lint
+lint:
+	pylint dast --max-line-length=120
+	mypy . --ignore-missing-imports
```
Comment (casellimarco) [inline Makefile:4]: why we ignore missing imports?

File: dast/from_ast.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        if key in ignored_props:
+            delattr(node, key)
+        else:
+            if isinstance(value, list):
+                for element in value:
+                    strip(element)
+            elif isinstance(value, ast.AST):
+                strip(value)
+            else:
+                pass
```
Comment (casellimarco) [inline dast/from_ast.py]: is pylint complaining if we nuke this two lines?

File: Makefile (hunk 2 above)
Comment (wplohrmann) [reply Makefile:4]: Not all packages include types unfortunately and they are errors with mypy. If you try to run `make lint` without it you'll see the errors are not very helpful and nothing we can do anything about

File: dast/from_ast.py (hunk 3 above)
Comment (wplohrmann) [reply dast/from_ast.py]: Ah no, that's just me 😅

File: dast/__main__.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+[difftool "dast"]
+        cmd = "python3 -m dast $LOCAL $REMOTE"
+[difftool]
+        prompt = false
+```
+You can then run it with `git difftool -t dast`
+"""
+import ast
+import sys
+from functools import partial
```
Comment (casellimarco) [inline dast/__main__.py:16]: :feelsgood:

File: dast/from_ast.py (hunk 3 above)
Comment (casellimarco) [reply dast/from_ast.py]: Can we remove it then? :smile:

File: Makefile (hunk 2 above)
Comment (casellimarco) [reply Makefile:4]: :pain:

File: dast/from_ast.py (hunk 3 above)
Comment (wplohrmann) [inline dast/from_ast.py]:
```suggestion
```

**Review by casellimarco — COMMENTED** [review]
Beautiful

**Pull request merged**